capstone/capbrowse/views.py

```diff
@@ -0,0 +1,57 @@
+from django.shortcuts import render
+from capdb.models import Jurisdiction, Court, Reporter
+from collections import OrderedDict
+
+
+# Create your views here.
+
+def view_case(request, case_id):
+    return render(request, "view_case.html", {'case_id': case_id})
+
+
+def view_jurisdiction(request, jurisdiction_id):
+    jurisdiction = Jurisdiction.objects.get(pk=jurisdiction_id)
```

Comment:
This line is better off using `get_object_or_404` so it doesn't throw a 500 with an unknown jurisdiction. Same for subsequent functions.

Reply:
Done.